README.md

```diff
@@ -186,6 +188,7 @@ You can find a list of Discord API libraries here. Libraries are sorted alphabet
 
 ### [TypeScript](https://www.typescriptlang.org "TypeScript")
 
+- [@discordjs/voice](https://github.com/discordjs/voice "discordjs/voice")<sup>[2]</sup>
```

Comment:
I know its written in typescript but wouldn't it be better suited in the JavaScript category as it can be used for both JS and TS.

Reply:
Sure thing